tools/benchmark/README.md

```diff
@@ -2,8 +2,20 @@
 
 `etcd/tools/benchmark` is the official benchmarking tool for etcd clusters.
 
+## Download
+To get `etcd` from the `master` branch via `go get`:
+```sh
+go get github.com/coreos/etcd/tools/benchmark
```

Comment:
My understanding is that 'go get' will perform download AND install.

Reply:
Thanks, how about now?